src/PokedexLayout/PokedexLayout.css

```diff
@@ -0,0 +1,136 @@
+.j-pokedexLayout-background {
+    background-color: #E54A28;
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.j-pokedexLayout-bottom {
```

Comment:
use nesting, so you need to repeat the class name
for more info, visit
https://sass-lang.com/documentation/style-rules

Reply:
Fixed